new-lamassu-admin/src/pages/TestingAddresses/TestingAddressesModal.js

```diff
@@ -0,0 +1,88 @@
+import { Box } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import { Formik, Form, Field } from 'formik'
+import * as R from 'ramda'
+import React from 'react'
+import * as Yup from 'yup'
+
+import ErrorMessage from 'src/components/ErrorMessage'
+import Modal from 'src/components/Modal'
+import { Link } from 'src/components/buttons'
+import { TextInput } from 'src/components/inputs/formik'
+import { H3 } from 'src/components/typography'
+
+import styles from './TestingAddresses.styles'
+const useStyles = makeStyles(styles)
+
+const TModal = ({ onClose, selectedCoin, addTestingAddress, errorMsg }) => {
+  const classes = useStyles()
+  const handleAddToTesting = address => {
+    if (selectedCoin.code === 'BCH' && !address.startsWith('bitcoincash:')) {
+      address = 'bitcoincash:' + address
+    }
+    addTestingAddress(selectedCoin.code, address)
+  }
+  const placeholderAddress = {
+    BTC: '1ADwinnimZKGgQ3dpyfoUZvJh4p1UWSSpD',
+    ETH: '0x71C7656EC7ab88b098defB751B7401B5f6d8976F',
+    LTC: 'LPKvbjwV1Kaksktzkr7TMK3FQtQEEe6Wqa',
+    DASH: 'XqQ7gU8eM76rEfey726cJpT2RGKyJyBrcn',
+    ZEC: 't1KGyyv24eL354C9gjveBGEe8Xz9UoPKvHR',
+    BCH: 'qrd6za97wm03lfyg82w0c9vqgc727rhemg5yd9k3dm',
+    USDT: '0x5754284f345afc66a98fbb0a0afe71e0f007b949',
+    XMR:
+      '888tNkZrPN6JsEgekjMnABU4TBzc2Dt29EPAvkRxbANsAnjyPbb3iQ1YBRk1UXcdRsiKc9dhwMVgN5S9cQUiyoogDavup3H'
+  }
```

Comment:
Since this is repeated from the blacklist page, could we just move the variable up to a constants file? Admin one should be in `src/utils/constants.js`